v23.1/cockroach-node.md

```diff
@@ -117,10 +117,12 @@ Flag | Description
 `--stats` | Show node disk usage details.
 `--timeout` | Set the duration of time that the subcommand is allowed to run before it returns an error and prints partial information. The timeout is specified with a suffix of `s` for seconds, `m` for minutes, and `h` for hours. If this flag is not set, the subcommand may hang.
 
-The `node decommission` subcommand also supports the following general flags:
+The `node decommission` subcommand also supports the following general flags. For more information, see `cockroach node decommission --help`.
 
 Flag | Description
 -----|------------
+`--checks` | <a name="decommission-checks"></a> Whether to perform a set of "decommissioning pre-flight checks". Possible values: `enabled`, `strict`, or `skip`. If `enabled`, CockroachDB will check if a node can successfully complete decommissioning given the current state of the cluster. If errors are detected that would result in the inability to complete node decommissioning, they will be printed to `STDERR` and the command will exit *without attempting to perform node decommissioning*. For more information, see [Remove nodes](node-shutdown.html?filters=decommission#remove-nodes).<br/><br/>**Default:** `enabled`
```

Comment:
Is the `strict` value not documented?

Reply:
it's very very complicated IMO.  I have a DM from Alex where there are many (again just IMO) words required to explain a scenario where the difference matters, the tl;dr: is that it can block a decommission that could have actually completed successfully and you have to think about like, the state of specific replicas and stuff

so I've opted to punt (for now) to the output of `cockroach node decommission --help` due to time constraint but also because really explaining the distinction between these flags would not be at all simple and it's not clear (yet, to me) that the difference is worth going into for majority of users.  and we can always add more doc of course

Reply:
oh and for completeness I should say we did discuss whether to doc this level of detail in that same DM convo and landed on let's punt for now since it could involve explaining some implementation details of how the allocator works that are subject to change (Alex please correct me if I'm misunderstanding!)

but for now at least I will merge this since At Least It's Something (tm)